boot/zephyr/boards/thingy91x_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,54 @@
+# MCUBoot settings
+CONFIG_BOOT_MAX_IMG_SECTORS=110
+
+# MCUboot serial recovery
+CONFIG_MCUBOOT_SERIAL=y
+
+# Disable Zephyr console
+CONFIG_LOG=n
+CONFIG_CONSOLE=n
+CONFIG_CONSOLE_HANDLER=n
+CONFIG_UART_CONSOLE=n
+
+# Serial
+CONFIG_SERIAL=y
+CONFIG_UART_NRFX=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_LINE_CTRL=y
+
+# MCUboot serial recovery
+CONFIG_GPIO=y
+CONFIG_MCUBOOT_SERIAL=y
+CONFIG_BOOT_SERIAL_CDC_ACM=y
+
+# Required by USB
+CONFIG_MULTITHREADING=y
+
+# USB
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="MCUBOOT"
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_COMPOSITE_DEVICE=y
+CONFIG_USB_MASS_STORAGE=n
+CONFIG_USB_DEVICE_MANUFACTURER="Nordic Semiconductor"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x520F
```

Comment:
`CONFIG_USB_DEVICE_PID` is the same as for thingy:91. Shouldn't the X have a unique PID?

Reply:
Maybe - the nrf53 board files are not used by default, mostly for brave people ready to overwrite the onboard programmer.